### PR TITLE
Add ability to set classes to list items

### PR DESCRIPTION
**Pull request by ksCookj**
Had a requirement for my project to individually add classes to list items in the dropdown. These minor changes solve this.

I can see this being useful to others.

### CODE REVIEW

File: src/dropdown/dropdown.component.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
     </li>
     <li *ngIf="settings.showCheckAll || settings.showUncheckAll" class="dropdown-divider divider"></li>
     <li class="dropdown-item" [ngStyle]="getItemStyle(option)" *ngFor="let option of options | searchFilter:searchFilterText"
-      (click)="!option.isLabel && setSelected($event, option)" [class.dropdown-header]="option.isLabel">
+      (click)="!option.isLabel && setSelected($event, option)" [class.dropdown-header]="option.isLabel" [ngClass]="option.classes ? option.classes : null">
```
Comment (nicholasguyett) [inline src/dropdown/dropdown.component.html]: What's the motivation behind converting an falsey value to null?

Unless I'm missing something, it should work without problem as just `[ngClass]="option.classes"`.  If it is needed, I'd at least prefer that you simplify it to `[ngClass]="option.classes || null"`.

Aside from that, LGTM.  Good job.

**Comment by ksCookj** [comment]
Removed ternary expression as suggested. Originally did not notice the class attribute will always be on each list item.

File: src/dropdown/dropdown.component.html (hunk 1 above)
Comment (ksCookj) [inline src/dropdown/dropdown.component.html]: see latest commit. Did not notice class attribute is always applied to each list item. Was using ternary to not add unused class attribute.

**Comment by nicholasguyett** [comment]
I'll manually test this and merge it today if I don't see any issues.

**Comment by nicholasguyett** [comment]
I'll manually test this later today and merge it in if there are no problems.  Thank you very much for your contribution.

I just need to verify that using `[class.someClass]` and `[ngClass]` doesn't clash with each other.  I don't think it does, but I'd hate to merge something in blind and then break it all...

**Comment by nicholasguyett** [comment]
Sorry for the delay.  Fixed the conflicts and merged in your branch.

**Pull request merged**